docs/SetupDatabaseForAzure.md

```diff
@@ -48,3 +48,24 @@ You can choose Flexible or Single server deployment for Azure Database for MySQL
 Note:-
 
 * The [Storage auto-grow](https://docs.microsoft.com/en-gb/azure/mysql/concepts-pricing-tiers#storage-auto-grow) feature automatically increases storage without impacting the workload and is enabled by default in server configuration while creating an Azure Database for MySQL instance.
+
+
+## Azure Database for PostgreSQL
+
+In this section, you will create an Azure Database for PostgreSQL.
+
+### Steps
+
+You can choose Flexible or Single server deployment for Azure Database for PostgreSQL, follow the [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/overview-postgres-choose-server-options) to learn more.
+* To create an Azure Database for PostgreSQL, 
```

Comment:
It'a an action, right? Why starting with `To create`? Should it be `Create an ...`?